test/definitions/test_helper.rb

```diff
@@ -84,6 +84,9 @@ def refute_scenario(filter)
 end
 
 TEST_DIR = File.dirname(__FILE__)
-ForemanMaintain.setup(
-  :config_file => File.join(TEST_DIR, 'config/foreman_maintain.yml.test')
-)
+CONFIG_FILE = File.join(TEST_DIR, 'config/foreman_maintain.yml.test').freeze
+STORAGE_DATA_FILE = '/lib/foreman-maintain/data_test.yml'.freeze
```

Comment:
I'll think about it once again. Maybe @iNecas might have a clue about what's going wrong here.